common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -8,7 +8,7 @@
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
     <fieldset ${describedby}>
       % if response_data['label']:
-        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${HTML(response_data['label'])}</legend>
```

Comment:
It would be great to add default escaping through the page default for all the templates you are concerned with and can fix.